src/tests/JIT/Directed/Convert/out_of_range_fp_to_int_conversions.cpp

```diff
@@ -18,6 +18,7 @@ typedef enum {
     CONVERT_SATURATING,
     CONVERT_NATIVECOMPILERBEHAVIOR,
```

Comment:
Is it useful to have the `CONVERT_NATIVECOMPILERBEHAVIOR` option as part of the test? I do not see it used anywhere.

Reply:
Its left from some investigation work, to help identify what was happening in the various systems. I left it in as it didn't really hurt anything, but I can easily remove if you'd like.